package.json

```diff
@@ -152,6 +152,7 @@
     "test:peril": "cd peril && yarn test",
     "test:update": "jest --updateSnapshot",
     "test:watch": "jest --watch",
+    "test:specific": "jest",
```

Comment:
Why this? You can also do `yarn jest` no need to have a specific command
```suggestion
```

Reply:
this was a mistake